### PR TITLE
homework 4 completed

### PR DESCRIPTION
**Pull request by vakal237078**

### CODE REVIEW

**Review by Davydovychyurii — APPROVED** [review]

File: HW4/src/isBigger.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function isBigger(a, b){
+    return a > b;
+}
+
+console.log(isBigger(9, 6));
+
```
Comment (Kiril99) [inline HW4/src/isBigger.js:6]: use console.log() in the functions

Comment (vakal237078) [reply HW4/src/isBigger.js:6]: Why?
it doesn’t affect how the code works!

**Review by Kiril99 — APPROVED** [review]

File: HW4/src/getClosestToZero.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const closest = (array) => array.sort( (a, b) => Math.abs(0 - a) - Math.abs(0 - b))[0];
+
+console.log(closest([-3, -2, 6, 8, -1, 1, 12]));
```
Comment (AgentPrus) [inline HW4/src/getClosestToZero.js]: what if I'll give you numbers instead of array are they work ?

File: HW4/src/getMin.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function getMin(a, ...b) {
```
Comment (AgentPrus) [inline HW4/src/getMin.js:1]: you can use arguments object

Comment (AgentPrus) [reply HW4/src/getMin.js:1]: Math.min(...arguments);

Comment (vakal237078) [reply HW4/src/getMin.js:1]: +

File: HW4/src/getClosestToZero.js (hunk 2 above)
Comment (vakal237078) [reply HW4/src/getClosestToZero.js]: +

**Pull request merged**